README.md

```diff
@@ -199,3 +212,9 @@ or for all the services
 ```bash
 mini-langstream dev build
 ```
+If you run into issues with the build passing locally but failing the style checks in the CI, you can run:
+```bash
+mvn license:format 
```

Comment:
I prefer to suggest to use the standard Maven phase "generate-sources"
./mvn generate-sources

This way in  the future we can change the plugins without having to care about this docs